Topics/starting-a-data-project.md

```diff
@@ -0,0 +1,254 @@
+# Starting a Data Project
+
+# What you will learn 
+In this guide, you'll learn to prepare yourself to start the project. 
+
+This will consist of learning how to look for sources that can help you solve the problem, building the project structure, build good documentation, secure your code, our data and more!
+
+In this guide we'll make heavy use of an awesome best practices collection from the [PLOS | Public Library of Science](https://www.plos.org/). 
+
+Do you know the happiness when you find a _giant diamond cluster in Minecraft_? 
+
+If yes, you need to know that you've just found a fat one:
+take your time to explore the [**Ten Rules Collection**](https://collections.plos.org/ten-simple-rules).
+
+
+## Prerequisites
+All the previous Purgatorio guides.
+
+## Time to complete
+10 minutes.
+
+# Index
+- [Collect Information](#Hunting-for-Information)
+- [Building a Knowledge Tree](#Building-a-Knowledge-Tree)
+- [Choosing a Project Structure](#Choosing-a-Project-Structure)
+- [Reproducibility](#Reproducibility)
+- [Versioning](#Versioning)
+- [Documentation](#Documentation)
+- [Prepare to Fail](#Prepare-to-Fail)
+
+Let's dive right in!
+ 
+ 
+### Hunting for Information
+A vital phase when starting a project is to search for information that can help you. These can be of any kind:
+- Tutorials
+- Documentation
+- Existing projects
+- Research Papers
+- ...
+
+This guide will not teach you how to use [Google](https://www.google.com/), because you will already know if you are reading these lines :-). 
+
+By the way, probably not everyone knows about [**these Google tricks**](https://smallbiztrends.com/2019/03/google-tricks.html)...
+
+But there are a few tips that can come in handy when you start "amassing" knowledge that will then come in handy.
+
+#### Understand what you're looking for
+The first thing to do when dealing with a new problem is to make sure you're looking for the right things. Are you sure the problem is called that for example? Image segmentation is different from image classification! [This Google guide](https://developers.google.com/machine-learning/problem-framing/cases) can help you be sure of the name of your problem.
+
+This may seem trivial, but many useful resources are not found because the correct keywords are not typed into the search engine.
```

Comment:
👍

Reply:
I would add here somewhere asking the right questions and also using the [5-why's method](https://en.wikipedia.org/wiki/Five_whys).

Reply:
I ve spoken bout this in the previous guide (frame the problem). do you think that s needed here too?